src/net_processing.h

```diff
@@ -72,6 +72,11 @@ struct Peer {
 
     /** This peer's reported block height when we connected */
     std::atomic<int> m_starting_height{-1};
+    /** The final block hash in an INV message. When the peer requests this
```

Comment:
b5b27ea052905ff7f26a962baa961fcc762c5ee0

I don't think this is true either. `m_hash_continue` is a purely internal helper, never sent over the wire directly.  Instead, the tip hash is sent to trigger the peer to ask further.

Reply:
I've updated the comment to be "The final block hash that we sent in an `inv` message to this peer." Let me know if you have suggestions for better wording.

Reply:
Thanks. Looks good now.